src/rebar_file_utils.erl

```diff
@@ -86,6 +88,36 @@ consult_config(State, Filename) ->
 format_error({bad_term_file, AppFile, Reason}) ->
     io_lib:format("Error reading file ~s: ~s", [AppFile, file:format_error(Reason)]).
 
+-spec symlink_or_create(Source, Target) -> Result when
+    Source :: file:filename(),
+    Target :: file:filename(),
+    Result :: Ok | Error,
+    Ok :: ok,
+    Error :: {error, string()}.
+symlink_or_create(Source, Target) ->
+    SourceExists = ec_file:is_dir(Source) orelse ec_file:is_symlink(Source),
```

Comment:
it's just for directories, but i guess it would be easy to account for files also